examples/max_clique/max_clique.ipynb

```diff
@@ -495,23 +503,27 @@
     "    # explicit constraint function defined by given graph\n",
     "    explicit_constraint = lambda x: old_constraint(x, edges)\n",
     "    explicit_qubo, offset = SamplingCompiler.generate_qubo(\n",
-    "    cost_function, explicit_constraint, n, penalty_method=\"sum\"\n",
+    "    cost_function, explicit_constraint, n, penalty_method=\"sum\", use_multiprocessing=False\n",
     ")\n",
     "stop_exp = timer()"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+<<<<<<< HEAD
```

Comment:
@lpodl There are traces of commit conflict not being solved properly  from this line down to line 517. Please clear your notebook outputs before commit, it will avoid `execuction_count` fields creating issues. In current state, the notebook is unreadable.

Reply:
copy-pasted the notebook from before the merge happened and cleared all outputs